java/ql/lib/semmle/code/xml/AndroidManifest.qll

```diff
@@ -67,6 +67,26 @@ class AndroidApplicationXmlElement extends XMLElement {
       attr.getValue() = "true"
     )
   }
+
+  /**
+   * Holds if this application element enables the `android:allowBackup` attribute.
+   *
+   * https://developer.android.com/guide/topics/data/autobackup
+   */
+  predicate allowsBackup() {
```

Comment:
As discussed elsewhere, this looks fine, but maybe it would be simpler to just hold if an `allowBackup` attribute with value `"false"` doesn't exist.

Reply:
Good idea. Implemented.